js/main.js

```diff
@@ -1,6 +1,18 @@
 var languages = ['en', 'fr'];
 var langIdx = 0;
 
+Array.prototype.rotatel = function() {
+    var t = this.shift();
+    this.push(t);
+    return t;
+}
+
```

Comment:
Avoid changing prototypes, especially of objects provided standard. For one argument see:

https://stackoverflow.com/questions/39579042/change-array-prototype-in-node-js

Also, avoid single letter variable for reasons of readability. What does 't' represent? The only real exception is for loops, for indicating the index.